spyder/utils/help/tutorial.rst

```diff
@@ -761,100 +732,99 @@ Debugging
 Line by line step execution of code
 -----------------------------------
 
-Activating the debug mode (with the ``Debug > Debug`` menu option or Ctrl+F5)
-starts the Python debugger (Pdb) if the Python console is active, or the IPython
-debugger (ipdb) if the IPython console is active. After doing that, the
-Editor pane will highlight the line that is about to be executed, and the
+Activating the debug mode (with the ``Debug > Debug`` menu option or ``Ctrl+F5)
+the IPython debugger ``ipdb``. The Editor pane will then
```

Comment:
"starts" is missing before "the IPython debugger".

Reply:
Yep, silly mistake, and its already fixed in #7659 .